modules/ROOT/partials/couchbase-operator-2.7.1-release-notes.adoc

```diff
@@ -0,0 +1,39 @@
+[#release-271]
+== Release 2.7.1 (August 2025)
```

Comment:
Can we please have this consistent across all the Release Notes - if the standard is to have dates alongside the version then please add them to the existing versions.